std/range/package.d

```diff
@@ -236,27 +236,33 @@ public import std.typecons : Flag, Yes, No;
 
 import std.internal.attributes : betterC;
 import std.meta : allSatisfy, staticMap;
-import std.traits : CommonType, isCallable, isFloatingPoint, isIntegral,
-    isPointer, isSomeFunction, isStaticArray, Unqual, isInstanceOf;
+import std.traits : ReturnType;
+import std.traits : CommonType;
+import std.traits : isFloatingPoint, isIntegral, isPointer, isStaticArray;
+import std.traits : isCallable, isSomeFunction, isInstanceOf;
+import std.traits : Unqual;
```

Comment:
There is no need for multiple import statements of the same module.

Reply:
The way it was before was harder to read IMO.

Reply:
This change does not follow the usual guidelines in phobos. I suggest undoing it.